process_metrics_linux.go

```diff
@@ -123,6 +156,10 @@ func writeIOMetrics(w io.Writer) {
 			writeBytes = getInt(s)
 		}
 	}
+
+	fmt.Fprintf(w, "process_io_stats_read_errors_total{reason=\"not_found\"} %d\n", atomic.LoadInt64(&processSelfIONotFoundErrorCount))
```

Comment:
I don't like the idea of these metrics, frankly)
they would matter only if the error is recoverable. But if not - no point in incrementing them. You can have a metric set to 0 or 1 in init func depending on whether /proc/self/io is accessible. But that is also a bit of extra - who'd look for these metrics anyway?